variables.tf

```diff
@@ -84,3 +91,26 @@ variable "infrastructure_admin_ssh_private_key_filepath" {
   type = string
   default = "~/.ssh/id_rsa"
 }
+variable "aap_red_hat_username" {
+  description = "The RedHat Account name"
```

Comment:
Change to "Red Hat account"

Do we need to be more descriptive of what account?